Backend/BikesRentalServer.DataAccess/Repositories/Abstract/IMalfunctionsRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BikesRentalServer.Models;
```

Comment:
ja z moim upośledzeniem klasycznie ładnie poproszę o posortowanie usingów, ale możesz mnie olać (i tak to zrobię w końcu ;p)